### PR TITLE
FE-126: Views UI Layout update

### PR DESCRIPTION
**Pull request by palashkaria**
### Scope & Purpose

This updates the views UI layout to a split pane
1. Left pane has an accordion with forms/inputs
2. Right pane has JSON editor

- [x] :pizza: New feature

### Checklist

- [x] Tests
  - [x] Manually tested
- [x] :book: CHANGELOG entry made

#### Related Information

- [x] Jira ticket: https://arangodb.atlassian.net/browse/FE-126

Screenshot: 
image

### CODE REVIEW

**Review by ajurdak — APPROVED** [review]
LGTM.

**Review by adityamukho — COMMENTED** [review]
Can't add new links from the web form. Auto-complete shows no options in list, and free-form names can't be entered.

JSON form works fine.

**Review by adityamukho — COMMENTED** [review]
One more point: Please make the JSON form read-only for users with no write access to the view.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/forms/ConsolidationPolicyForm.tsx [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+
 const BytesAccumConsolidationPolicyForm = ({
-                                             formState,
-                                             dispatch,
-                                             disabled
-                                           }: FormProps<BytesAccumConsolidationPolicy>) => {
+  formState,
+  dispatch,
+  disabled
+}: Omit<FormProps<BytesAccumConsolidationPolicy>, "dispatch"> & {
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/forms/ConsolidationPolicyForm.tsx:15]: Please define the prop type above to make it more readable.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/forms/ConsolidationPolicyForm.tsx [hunk 2]
```diff
@@ -27,17 +30,19 @@ const BytesAccumConsolidationPolicyForm = ({
...
 
 const TierConsolidationPolicyForm = ({
-                                       formState,
-                                       dispatch,
-                                       disabled
-                                     }: FormProps<TierConsolidationPolicy>) => {
+  formState,
+  dispatch,
+  disabled
+}: Omit<FormProps<TierConsolidationPolicy>, "dispatch"> & {
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/forms/ConsolidationPolicyForm.tsx:43]: Please define a type for the prop.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/forms/ConsolidationPolicyForm.tsx [hunk 3]
```diff
@@ -110,14 +115,20 @@ const TierConsolidationPolicyForm = ({
...
     </tr>
   </>;
 };
 
-const ConsolidationPolicyForm = ({ formState, dispatch, disabled }: FormProps<ViewProperties>) => {
+const ConsolidationPolicyForm = ({
+  formState,
+  dispatch,
+  disabled
+}: Omit<FormProps<ViewProperties>, "dispatch"> & {
```
Comment (adityamukho) [inline js/apps/system/_admin/aardvark/APP/react/src/views/views/forms/ConsolidationPolicyForm.tsx:129]: Prop type.

**Review by adityamukho — COMMENTED** [review]
Left some comments inline.

**Comment by palashkaria** [comment]
> Can't add new links from the web form. Auto-complete shows no options in list, and free-form names can't be entered.
> 
> JSON form works fine.

This was a bug in SWR which was preventing it from making the API calls. Found a fix for now, but needs to be investigated further. Will do that when using SWR next

The temp fix: https://github.com/arangodb/arangodb/pull/18220/commits/9ad18ad84696dde89a96d925d6048440d101e0cd

**Comment by palashkaria** [comment]
> One more point: Please make the JSON form read-only for users with no write access to the view.

As discussed, this was a product requirement (to keep the editable JSON field in read-only mode too), and it works fine as we remove the 'save' button in these cases

**Review by adityamukho — APPROVED** [review]
LGTM

**Pull request merged**